app/assets/stylesheets/govuk_publishing_components/components/_layout-for-public.scss

```diff
@@ -0,0 +1,4 @@
+.gem-c-layout-for-public {
+  @include govuk-font(16);
```

Comment:
Don't set a font size on the `html` or `body` – in a non-compatibility mode world all of the font sizes are defined in rem, but we rely on the user being able to change the root font-size in order to resize the text without zooming.

Also, because `govuk-font` uses the responsive typography scale, this'd result in the root font size being 14px on mobile, which means on mobile all of the text would be 14/16ths of what it should be.